### PR TITLE
doc: add how to series

### PR DESCRIPTION
**Pull request by xiaoyifang**

### CODE REVIEW

**Comment by shenlebantongying** [comment]
I think we should remove the howto folder, and move all of them to website to avoid unnecessary split.

For people with almost no computer background, md files within a git repo is weird.

The development related doc can be put under "Contributor Guides "

File: website/mkdocs.yml [hunk 1]
```diff
@@ -45,4 +45,13 @@ nav:
...
     - Developer: developer.md
+    - How to:
+       - how to build and debug with VS2019: "how to build and debug with VS2019.md"
+       - how to compile with cmake: "how to compile with cmake.md"
+       - how to connect with anki: "how to connect with anki.md"
+       - how to customize the opencc: "how to customize the opencc.md"
+       - how to find out the latest qt version and module in github qt action: "how to find out the latest qt version and module in github qt action.md"
+       - how to use .clang-format to format the code: "how to use .clang-format to format the code.md"
+       - how to use breadpad crash analysis: "how to use breadpad crash analysis.md"
+       - How to use ocr together with GoldenDict: "How to use ocr together with GoldenDict.md"
```
Comment (shenlebantongying) [inline website/mkdocs.yml:56]: "how to connect with anki" can be merged into "special usages -> anki integration"

"how to ocr" -> "special usages -> OCR integration"

seems better

Comment (xiaoyifang) [reply website/mkdocs.yml]: done

**Comment by shenlebantongying** [comment]
Not so important, but the titles can be shortened, so that table of content can be shorter
```
    - How to:
       - Build and debug with VS2019: "how to build and debug with VS2019.md"
       - Compile with cmake: "how to compile with cmake.md"
       - Customize the opencc: "how to customize the opencc.md"
       - Qt version and github action: "how to find out the latest qt version and module in github qt action.md"
       - Use .clang-format: "how to use .clang-format to format the code.md"
       - Breadpad crash analysis: "how to use breadpad crash analysis.md"
```

**Comment by shenlebantongying** [comment]
One last problem and this can be merged,

The url format is wrong, should be path without quote:

```
    - How to:
       - Build and debug with VS2019: howto/how to build and debug with VS2019.md
       - Compile with cmake: howto/how to compile with cmake.md
       - Customize the opencc: howto/how to customize the opencc.md
       - Qt version and github action: howto/how to find out the latest qt version and module in github qt action.md
       - Use .clang-format: howto/how to use .clang-format to format the code.md
       - Breadpad crash analysis: howto/how to use breadpad crash analysis.md
```

Right now, it will 404
image

**Comment by xiaoyifang** [comment]
> One last problem and this can be merged,
> 
> The url format is wrong, should be path without quote:
> 
> ```
>     - How to:
>        - Build and debug with VS2019: howto/how to build and debug with VS2019.md
>        - Compile with cmake: howto/how to compile with cmake.md
>        - Customize the opencc: howto/how to customize the opencc.md
>        - Qt version and github action: howto/how to find out the latest qt version and module in github qt action.md
>        - Use .clang-format: howto/how to use .clang-format to format the code.md
>        - Breadpad crash analysis: howto/how to use breadpad crash analysis.md
> ```
> 
> Right now, it will 404 image

seems I have missed the folder `howto`

File: website/mkdocs.yml [hunk 2]
```diff
@@ -42,7 +42,15 @@ nav:
       - Portable Mode: topic_portablemode.md
       - Custom transliteration: topic_transliteration.md
       - Customize Dictionary Name: custom_dictionary_name.md
+      - Ocr Integration: "How to use ocr together with GoldenDict.md"
```
Comment (shenlebantongying) [inline website/mkdocs.yml:45]: `- Ocr Integration: howto/How to use ocr together with GoldenDict.md`

`:)`

**Comment by shenlebantongying** [comment]
`how to build ffmpeg for visual studio.md` seems missed

**Comment by xiaoyifang** [comment]
added now

**Comment by shenlebantongying** [comment]
The `- Ocr Integration: "How to use ocr together with GoldenDict.md"`'s link is also wrong.

**Comment by shenlebantongying** [comment]
I mean the ocr's path is wrong should be `howto/How to use ocr together with GoldenDict.md` insteal of just `"How to use ocr together with GoldenDict.md"``:)`

**Review by shenlebantongying — APPROVED** [review]

**Pull request merged**